.travis.yml

```diff
@@ -34,7 +34,10 @@ script:
   - npm run cov:instrument
   - npm test --silent
   - npm run cov:report
-  - npm run test:release
+  - npm run test:artifacts
+  - if [ $TRAVIS_PULL_REQUEST_BRANCH == "" ]; then
```

Comment:
One other thing: can you also check that `travis tag` != `master`? That way we'll only run this when someone starts a release (the release process on GitHub creates a new branch).